orte/orted/orted_submit.c

```diff
@@ -589,6 +584,13 @@ int orte_submit_init(int argc, char *argv[],
          * orterun
          */
         orte_launch_environ = opal_argv_copy(environ);
+        /* clear params from the environment so our children
+         * don't pick them up */
+        opal_unsetenv(OPAL_MCA_PREFIX"ess", &orte_launch_environ);
+        opal_unsetenv(OPAL_MCA_PREFIX"pmix", &orte_launch_environ);
+        /* clear any install_prefix that might have been set so
+         * we don't cause our children to falsely generate warnings */
+        opal_unsetenv("PMIX_INSTALL_PREFIX", &orte_launch_environ);
```

Comment:
That's fine - it was being cleared to avoid conflicts on the internal version if OPAL_PREFIX was set and we then set PMIX_INSTALL_PREFIX in the daemon, thus delivering both to the child.